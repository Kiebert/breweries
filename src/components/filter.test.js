import React from 'react';
import { shallow } from 'enzyme';
import Filter from './filter';

describe('<Filter />', () => {
    it('renders correctly', () => {


        const component = shallow(
            <Filter />
        );
        expect(component).toMatchSnapshot();

    });

});