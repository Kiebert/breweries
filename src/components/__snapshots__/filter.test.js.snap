// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Filter /> renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="Filter"
  >
    <div
      className="Filter-Introduction"
    >
      Fill in a name, zipcode, address or city to find your brewery:
    </div>
    <input
      className="Filter-Input"
      onChange={null}
      placeholder="Type here.."
      type="text"
    />
  </div>,
  "nodes": Array [
    <div
      className="Filter"
    >
      <div
        className="Filter-Introduction"
      >
        Fill in a name, zipcode, address or city to find your brewery:
      </div>
      <input
        className="Filter-Input"
        onChange={null}
        placeholder="Type here.."
        type="text"
      />
    </div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Filter
        onChange={null}
      />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": Filter {
        "_reactInternalInstance": [Circular],
        "clearInput": [Function],
        "context": Object {},
        "props": Object {
          "onChange": null,
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="Filter"
        >
          <div
            className="Filter-Introduction"
          >
            Fill in a name, zipcode, address or city to find your brewery:
          </div>
          <input
            className="Filter-Input"
            onChange={null}
            placeholder="Type here.."
            type="text"
          />
        </div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="Filter"
        >
          <div
            className="Filter-Introduction"
          >
            Fill in a name, zipcode, address or city to find your brewery:
          </div>
          <input
            className="Filter-Input"
            onChange={null}
            placeholder="Type here.."
            type="text"
          />
        </div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Filter
    onChange={null}
  />,
}
`;
